sui/src/sui_commands.rs

```diff
@@ -554,9 +554,9 @@ pub async fn make_authority(
     // Spawn a consensus listener. It listen for consensus outputs and notifies the
     // authority server when a sequenced transaction is ready for execution.
     ConsensusListener::spawn(
-        authority_state.clone(),
         rx_sui_to_consensus,
         rx_consensus_to_sui,
+        /* max_pending_transactions */ 10_000,
```

Comment:
What is the implication of this? Make a const and document what it does?

Reply:
adding a comment